tracer/build/_build/docker/gitlab/requirements.txt

```diff
@@ -0,0 +1,19 @@
+# python development requirements for the Datadog Agent
```

Comment:
rather than copy/pasting the whole requirements file, this file can just be a redirect to the `datadog-agent-buildimages` requirements file so it's always up-to-date.

Reply:
That's good in theory, but we don't currently automatically rebuild this image anyway, so it won't _automatically_ be up to date, and it means our pipeline would break if the file moved (for example), so it might be best to just add a link to the file here and copy-past the `datadog-agent-buildimages` one in for now 🤔